ansiblelint/rules/RestrictCommandsCheck.py

```diff
@@ -0,0 +1,13 @@
+from ansiblelint import AnsibleLintRule
+
+class RestrictCommandsCheck(AnsibleLintRule):
+  id = '702'
+  shortdesc = 'commands like (awk/cut/set) should not be use'
+  description = 'Certain commands like awk/cut/set should not be use'
```

Comment:
Warn against usage of bash utilities like aw/sed/cut/grep

Reply:
Done